src/common/helpers/requestMetaData/approvedIdentifiersTableParser.ts

```diff
@@ -0,0 +1,91 @@
+import fs from "fs";
+import path from "path";
+import https from "https";
+
+main();
+
+async function main() {
+  // first download the table of approved identifiers from our docs
+  const url =
+    "https://raw.githubusercontent.com/UMAprotocol/docs/master/docs/uma-tokenholders/approved-price-identifiers.md";
+  const markdownPath = path.join(__dirname, "./approvedIdentifiersTable.md");
+  const jsonPath = path.join(__dirname, "./approvedIdentifiersTable.json");
+
+  await downloadFile(url, markdownPath);
+
+  // read and parse the markdown file from GitHub
+  const contents = fs.readFileSync(markdownPath, "utf-8");
+  const lines = contents
+    .split("\n")
+    // ignore the first 8 lines (headmatter and table headers)
+    .slice(8);
+  const parsedLines = JSON.stringify(lines.map(parseLine));
```

Comment:
nit: you can make this output formatted json by adding these args:

```suggestion
  const parsedLines = JSON.stringify(lines.map(parseLine), undefined, 2);
```

Reply:
This is awesome! Would definitely have done this if I had known about it. Will update 👍